doc/classes/InputEvent.xml

```diff
@@ -89,7 +89,7 @@
 			<param index="0" name="event" type="InputEvent" />
 			<param index="1" name="exact_match" type="bool" default="true" />
 			<description>
-				Returns [code]true[/code] if the specified [param event] matches this event. Only valid for action events i.e key ([InputEventKey]), button ([InputEventMouseButton] or [InputEventJoypadButton]), axis [InputEventJoypadMotion] or action ([InputEventAction]) events.
+				Returns [code]true[/code] if the specified [param event] matches this event. Only valid for action events i.e key ([InputEventKey]), button ([InputEventMouseButton] or [InputEventJoypadButton]), axis [InputEventJoypadMotion] or action ([InputEventAction]) events. Does not consider event states (such as being pressed, released, echoed, or canceled). 
```

Comment:
```suggestion
				Returns [code]true[/code] if the specified [param event] matches this event. Only valid for action events i.e key ([InputEventKey]), button ([InputEventMouseButton] or [InputEventJoypadButton]), axis [InputEventJoypadMotion] or action ([InputEventAction]) events.
				[b]Note:[/b] Only considers the event configuration, not state information like [method is_pressed], [method is_released], [method is_echo], or [method is_canceled].
```
I think reads better

Reply:
I did consider putting it into a Note, but I was not sure about the guidelines on Notes. So, I choose to add it above. When would adding notes be appropriate for the documentation?

Reply:
I'd say this is a prime example where we draw attention to some special detail or something people might not expect